app/inpututils.cpp

```diff
@@ -78,13 +78,29 @@ bool InputUtils::copyFile( const QString &srcPath, const QString &dstPath )
     modSrcPath = modSrcPath.replace( "file://", "" );
   }
 
+  QFileInfo fi( dstPath );
+  QDir dir = fi.absoluteDir();
+  if ( !dir.exists() )
+  {
+    if ( !dir.mkpath( dir.path() ) )
+    {
+      return false;
+    }
+  }
+
   // https://github.com/lutraconsulting/input/issues/418
   // does not work for iOS files with format
   // file:assets-library://asset/asset.PNG%3Fid=A53AB989-6354-433A-9CB9-958179B7C14D&ext=PNG
 
   return QFile::copy( modSrcPath, dstPath );
 }
 
+bool InputUtils::createDirectory( const QString &path )
+{
+  QDir dir;
+  return dir.mkpath( path );
```

Comment:
add `if ( !dir.exists() )`

Reply:
Not sure this is really needed, as `mkpath` will return `true` if path already exists.

Reply:
ok